packages/app/src/domain/topical/mini-trend-tile-layout.tsx

```diff
@@ -0,0 +1,23 @@
+import { Children, ReactNode } from 'react';
+import { Box } from '~/components-styled/base';
+import { Tile } from '~/components-styled/tile';
+
+type MiniTrendTileLayoutProps = {
+  children: ReactNode;
+};
+
+export function MiniTrendTileLayout({ children }: MiniTrendTileLayoutProps) {
+  const miniTrendTiles = Children.toArray(children);
```

Comment:
nit-nitpick;

```suggestion
  const tiles = Children.toArray(children);
```

I think the parent components already provides enough context so tiles would suffice.